hardhat.config.js

```diff
@@ -14,7 +14,7 @@ module.exports =
     hardhat: {
       forking: {
         url: `${process.env.NODE_URL_MAINNET}`,
-        blockNumber: 11868228,
+        blockNumber: 11954282,
```

Comment:
Why did you have to change the block?

Reply:
There was some issue of a missing trie node when running the tests. I think the  Tellor contract wasn't deployed in the old block

Reply:
Actually the problem here is with out eth node. I need to figure out what is the issue, but until then can use alchemyapi or infura for the tests.